cli/node_account/forms.py

```diff
@@ -0,0 +1,18 @@
+"""
+Provide forms for command line interface's node account commands.
+"""
+from marshmallow import Schema
+
+from cli.generic.forms.fields import (
+    AccountAddressField,
+    NodeUrlField,
+)
+
+
+class GetNodeAccountInformationForm(Schema):
+    """
+    Get the node account information.
```

Comment:
`Get information about the node account by its address form.`

Reply:
@dmytrostriletskyi, done.